src/DDPlanarMeasLayer.cc

```diff
@@ -100,6 +104,17 @@ DDPlanarMeasLayer::DDPlanarMeasLayer(dd4hep::rec::ISurface* surf, Double_t   Bz,
     fSortingPolicy =  surf->distance( dd4hep::rec::Vector3D( 0.,0.,0.) )/dd4hep::mm   +  epsilon * count++ ;
   }
 
+  auto& detElement = static_cast<dd4hep::rec::Surface*>(surf)->detElement();
+  if(detElement.isValid()) {
```

Comment:
what does it mean when this condition is false?

Reply:
Yes.